FindmallocMC.cmake

```diff
@@ -97,44 +102,47 @@ if(mallocMC_ROOT_DIR)
     # mallocMC variables ########################################################
     #
     set(mallocMC_VERSION "${mallocMC_VERSION_MAJOR}.${mallocMC_VERSION_MINOR}.${mallocMC_VERSION_PATCH}")
-    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include)
+    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include CACHE PATH "mallocMC include pathes")
```

Comment:
Problems:
 - mallocMC_LIBRARIES is not cached, whereas mallocMC_INCLUDE_DIRS is. This is inconsistent
 - It is not possible to use `list(APPEND ...)` for mallocMC_INCLUDE_DIRS (list commands can not handle cached variables).